src/components/mobile/NavBarBottom.jsx

```
@@ -42,34 +41,43 @@ export default function NavBarBottom() {
           label="Learning"
           testId="bottomnav-learning"
           isActive={active.learning}
         />
+        {/* Search button opens page-local search overlay via custom event */}
+        <button
+          type="button"
+          onClick={() => {
+            const page = pathname.startsWith('/staff/my-brands')
+              ? 'my_brands'
+              : (pathname.startsWith('/staff/learning') ? 'learning' : 'community');
+            try { window.dispatchEvent(new CustomEvent('en:openMobileSearch', { detail: { page } })); } catch (err) { /* no-op */ }
```

ESLint

[error] 'err' is defined but never used.

(no-unused-vars)